src/itest/kotlin/nl/info/zac/itest/ZaakKoppelenRestServiceTest.kt

```diff
@@ -114,9 +122,7 @@ class ZaakKoppelenRestServiceTest : BehaviorSpec({
             val response = itestHttpClient.performGetRequest(
                 url = "$ZAC_API_URI/zaken/gekoppelde-zaken/$zaakProductaanvraag1Uuid/zoek-koppelbare-zaken" +
                     "?zoekZaakIdentifier=$ZOEK_ZAAK_IDENTIFIER" +
-                    "&relationType=DEELZAAK" +
-                    "&rows=$ROWS_DEFAULT" +
-                    "&page=$PAGE_DEFAULT"
+                    "&relationType=DEELZAAK"
```

Comment:
Ah I see, here you perform the query without the `rows` and `page` request parameters to test calling the endpoint without these params right? I would make that clear in the test description. E.g. `Searching for a DEELZAAK linkable zaken with 'ZAAK-2000' as zaak identifier and without the 'rows' and 'page' request parameters".